src/vm/eventpipebuffermanager.h

```diff
@@ -100,6 +122,24 @@ class EventPipeBufferManager
     // Otherwise, if a stack trace is needed, one will be automatically collected.
     bool WriteEvent(Thread *pThread, EventPipeSession &session, EventPipeEvent &event, EventPipeEventPayload &payload, LPCGUID pActivityId, LPCGUID pRelatedActivityId, Thread *pEventThread = NULL, StackContents *pStack = NULL);
 
+    // Ends the suspension period created by SuspendWriteEvent(). After this call returns WriteEvent()
+    // can again be called succesfully, new BufferLists and Buffers may be allocated.
+    // The caller is required to synchronize all calls to SuspendWriteEvent() and ResumeWriteEvent()
+    void ResumeWriteEvent();
+
+    // From the time this function returns until ResumeWriteEvent() is called a suspended state will 
+    // be in effect that blocks all WriteEvent activity. All existing buffers will be in the 
+    // PENDING_FLUSH state and no new EventPipeBuffers or EventPipeBufferLists can be created. Calls to 
```

Comment:
```suggestion
    // READ_ONLY state and no new EventPipeBuffers or EventPipeBufferLists can be created. Calls to 
```

A relic of some earlier naming I had